src/transaction/makeFulfillment.js

```diff
@@ -0,0 +1,21 @@
+export default function makeFulfillment(issuers = []) {
+    if (issuers.length === 1) {
+        const fulfillment = {
+            type: 'ed25519-sha-256',
+            public_keys: issuers
+        }
+        return fulfillment
+    } else {
+        const subcdts = []
```

Comment:
`subcdts` is not an appreviation we commonly use in the code base.
I suggest to rename to a word we also use in other implementations.